core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -2221,6 +2221,18 @@ private CarbonCommonConstants() {
    */
   public static final String FACT_FILE_UPDATED = "update";
 
+  /**
+   * Configured property to enable/disable Load Failed Segments in SI Table during Load command
+   */
+  @CarbonProperty
+  public static final String CARBON_LOAD_SI_REPAIR = "carbon.load.si.repair";
+
+  /**
+   * Default value for Load Failed segments in SI table during load command property
+   */
+  @CarbonProperty
```

Comment:
This is just a default value. Not the property. Please remove `@CarbonProperty`annotation